scrapyd/sqlite.py

```diff
@@ -82,7 +83,7 @@ class JsonSqlitePriorityQueue(object):
     """SQLite priority queue. It relies on SQLite concurrency support for
     providing atomic inter-process operations.
     """
-    queue_priority_map = {}
+    project_priority_map = {}
```

Comment:
This is like "master table" solution I was talking about
actually implemented as a singleton to share state between all instances of the queue class
and save us lot of io and cpu cycles,
right?

Reply:
Yes, and it's fast to find out the queue to pop in poll()
since project_priority_map is a dict like:
`{'project1': (0.0, -1561646348), 'project2': (1.0, -1561646349)}`

So, there's no need to introduce an actual "master table".